mockOSApi.v2/Data/DTO/ProcessDto.cs

```diff
@@ -0,0 +1,78 @@
+using mockOSApi.Models;
+using mockOSApi.Repository;
+
+namespace mockOSApi.DTO;
+
+public class PriorityClass()
+{
+    public int Prio { get; set; }
```

Comment:
If the Priority is going to have specific values, you should use an Enum instead.

Reply:
Will be an int value, not specified value